Tests/PubNubTests/Networking/Routers/ChannelGroupEndpointTests.swift

```diff
@@ -252,9 +252,10 @@ extension ChannelGroupsRouterTests {
     guard let sessions = try? MockURLSession.mockSession(for: ["maximumChannelCountExceeded_Message"]) else {
       return XCTFail("Could not create mock url session")
     }
+    
```

Comment:
Codacy found an issue: [Lines should not have trailing whitespace.](https://app.codacy.com/manual/PubNub/swift/pullRequest?prid=6302911)